### ISSUE TITLE
Crash after clicking "Back"

### ISSUE THREAD

**Issue by VitalyPeryatin** [AlexGladkov/Odyssey#64]
If you are on the start screen and click on the "Back" button, then the application crashes. Exception stack trace:

```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.finance_tracker.finance_tracker, PID: 25018
    java.util.NoSuchElementException: List is empty.
        at kotlin.collections.CollectionsKt__MutableCollectionsKt.removeLast(MutableCollections.kt:255)
        at ru.alexgladkov.odyssey.compose.RootController.removeTopScreen(RootController.kt:372)
        at ru.alexgladkov.odyssey.compose.RootController.popBackStack(RootController.kt:199)
        at ru.alexgladkov.odyssey.compose.controllers.MultiStackRootController.popBackStack(MultiStackRootController.kt:44)
        at ru.alexgladkov.odyssey.compose.RootController.popBackStack(RootController.kt:198)
        at ru.alexgladkov.odyssey.compose.RootController$setupBackPressedDispatcher$1.onBackPressed(RootController.kt:126)
        at ru.alexgladkov.odyssey.core.backpress.OnBackPressedDispatcher.onBackPressed(OnBackPressedDispatcher.kt:7)
        at ru.alexgladkov.odyssey.compose.extensions.RootController___LifeCycleKt$setupWithActivity$1.handleOnBackPressed(RootController + LifeCycle.kt:19)
        at androidx.activity.OnBackPressedDispatcher.onBackPressed(OnBackPressedDispatcher.java:192)
        at androidx.activity.ComponentActivity.onBackPressed(ComponentActivity.java:657)
        at android.app.Activity.onKeyUp(Activity.java:3707)
        at android.view.KeyEvent.dispatch(KeyEvent.java:2849)
        at android.app.Activity.dispatchKeyEvent(Activity.java:4024)
        at androidx.core.app.ComponentActivity.superDispatchKeyEvent(ComponentActivity.java:124)
        at androidx.core.view.KeyEventDispatcher.dispatchKeyEvent(KeyEventDispatcher.java:86)
        at androidx.core.app.ComponentActivity.dispatchKeyEvent(ComponentActivity.java:142)
        at com.android.internal.policy.DecorView.dispatchKeyEvent(DecorView.java:389)
        at android.view.ViewRootImpl$ViewPostImeInputStage.processKeyEvent(ViewRootImpl.java:5545)
        at android.view.ViewRootImpl$ViewPostImeInputStage.onProcess(ViewRootImpl.java:5413)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4918)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4971)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4937)
        at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5077)
        at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4945)
        at android.view.ViewRootImpl$AsyncInputStage.apply(ViewRootImpl.java:5134)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4918)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4971)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4937)
        at android.view.ViewRootImpl$InputStage.apply(ViewRootImpl.java:4945)
        at android.view.ViewRootImpl$InputStage.deliver(ViewRootImpl.java:4918)
        at android.view.ViewRootImpl$InputStage.onDeliverToNext(ViewRootImpl.java:4971)
        at android.view.ViewRootImpl$InputStage.forward(ViewRootImpl.java:4937)
        at android.view.ViewRootImpl$AsyncInputStage.forward(ViewRootImpl.java:5110)
        at android.view.ViewRootImpl$ImeInputStage.onFinishedInputEvent(ViewRootImpl.java:5273)
        at android.view.inputmethod.InputMethodManager$PendingEvent.run(InputMethodManager.java:3071)
        at android.view.inputmethod.InputMethodManager.invokeFinishedInputEventCallback(InputMethodManager.java:2614)
        at android.view.inputmethod.InputMethodManager.finishedInputEvent(InputMethodManager.java:2605)
        at android.view.inputmethod.InputMethodManager$ImeInputEventSender.onInputEventFinished(InputMethodManager.java:3048)
        at android.view.InputEventSender.dispatchInputEventFinished(InputEventSender.java:143)
        at android.os.MessageQueue.nativePollOnce(Native Method)
        at android.os.MessageQueue.next(MessageQueue.java:336)
        at android.os.Looper.loop(Looper.java:181)
        at android.app.ActivityThread.main(ActivityThread.java:7562)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
```

**Comment by mironoff2007** [comment]
Is this issue actual? I can't reproduce this issue.

**Comment by AlexGladkov** [comment]
Fixed in 1.3
